package.json

```diff
@@ -23,7 +23,7 @@
     "build": "NODE_ENV=production webpack",
     "build:watch": "NODE_ENV=production webpack --watch",
     "start": "node example.js",
-    "start:dev": "yarn build:watch && yarn start",
+    "start:dev": "yarn build:watch & yarn start",
```

Comment:
this'll start the process in the background and might prove hard to kill - is that what you want?
consider `concurrently` lib instead

Reply:
https://github.com/vcapretz/bull-board/pull/10/commits/3d1a9c2c77e2583d922b1329daf344d71c9e13da